src/server/src/main/java/io/cassandrareaper/management/http/HttpCassandraManagementProxy.java

```diff
@@ -255,8 +292,15 @@ public Set<Table> getTablesForKeyspace(String keyspace) throws ReaperException {
   }
 
   @Override
-  public int getPendingCompactions() throws JMException {
-    return 1; // TODO: implement me.
+  public int getPendingCompactions() throws ReaperException {
+    List<GenericMetric> tpStatsMetrics = metricsProxy.collectTpPendingTasks();
+    for (GenericMetric metric : tpStatsMetrics) {
+      if (metric.getMetricName().equals("PendingTasks")
+          && metric.getMetricScope().equals("CompactionExecutor")) {
+        return (int) metric.getValue();
+      }
+    }
+    return 0;
```

Comment:
Issue: I assume this only gets hit if the tpStatsMetrics doesn't have a `CompactionExecutor` item with scope = `PendingTasks`. I think in that case maybe this is an error and we should log a warning and throw, what do you think?

Reply:
Fair enough, I thought I was reproducing the behavior of the JMX implementation, but it turns out we re-throw JMExceptions as ReaperExceptions, which should include the case where we can't find the metric we're looking for. I'll do the same here and throw a ReaperException if the metric can't be found.

Reply:
I don't know if it is just the diff issues I'm seeing, but this doesn't appear to have been updated to throw an exception?